ui/src/message_popup/message_popup_page.jsx

```diff
@@ -172,16 +186,25 @@ export default React.createClass({
             labelPosition="right"
             toggled={this.state.shouldShowStudentCards}
             onToggle={this.onStudentCardsToggled} />
-          <Toggle
-            label="With feeback and revision"
-            labelPosition="right"
-            toggled={false}
-            disabled={true}  />
-          <Toggle
-            label="With hints available"
-            labelPosition="right"
-            toggled={this.state.allowedToToggleHint}
-            onToggle={this.onHintsOptionToggled} />
+          <div style={{margin: 10}}><Divider /></div>
+          <RadioButtonGroup name="helpOptions" defaultSelected="none" onChange={this.onHelpToggled}>
```

Comment:
Do you notice anything strange if you select a radio button, and then start typing in the text area?  I think you may need to change from using `defaultSelected` to using `valueSelected`.

React components for user input often have two ways to change their value, [controlled and controlled](https://facebook.github.io/react/docs/forms.html#controlled-components).